NuGet.Config

```diff
@@ -10,6 +10,8 @@
       Used for azure-sdk-tools repo until issue https://github.com/Azure/azure-sdk-tools/issues/1329 is addressed
     -->
     <add key="azure-sdk-tools" value="https://azuresdkartifacts.blob.core.windows.net/azure-sdk-tools/index.json" />
+    <add key="azure-appservice" value="https://www.myget.org/F/azure-appservice/api/v3/index.json" /> <!-- Draft PR,  remove after WebJobs SDK is released-->
```

Comment:
Can be removed

Reply:
we still need this to build the PR -  Microsoft.Azure.WebJobs 3.0.35 haven't been released to official nuget yet. We can release 3.0.35 only after function host release with reference to 3.0.35.
This will be removed in future.